manage-applications.md

```diff
@@ -478,26 +478,42 @@ The most important part of an application version is the `published` field. If a
 
 Marking an application version as published can be achieved with the following command
 
-```bash
-$ amc application publish bcmap7u5nof07arqa2ag 1
-```
+    amc application publish bcmap7u5nof07arqa2ag 1
 
 Next to publishing an application version its already possible to revoke it
 
-```bash
-$ amc application revoke bcmap7u5nof07arqa2ag 1
-```
+    amc application revoke bcmap7u5nof07arqa2ag 1
 
 As consequence if an application only has a single published version and that version is revoked the application can't be used by any users anymore. AMS will still list the application but will mark it as not published as it has no published versions.
 
 Each version takes space on the LXD nodes. To free up space and remove old and unneeded version they can be individually removed with the only requirement that an application needs to have at least a single version at all time. Removing a specific application version is possible with the following command:
 
-```bash
-$ amc application delete --version=1 bcmap7u5nof07arqa2ag
-```
+    amc application delete --version=1 bcmap7u5nof07arqa2ag
 
 The `amc` command will ask for your approval before the version is removed as it may affect your users. If you want to bypass the check you can add the `--yes` flag to the command.
 
+## Disable Automatic Application Updates
+
+**since 1.11.0**
```

Comment:
```suggestion
*since 1.11.0*
```
That's how it is used in other topics. Any objection to changing it to "*Added in 1.11.0*"?